src/descriptor.rs

```diff
@@ -165,6 +165,18 @@ impl Descriptor {
         bytes
     }
 
+    /// Generates fixed bytes of payload of length specified by the generic parameter.
```

Comment:
```suggestion
    /// Generates fixed bytes of payload of length specified by the generic parameter.
    ///
```
empty line before header

Reply:
done in [036ce0e](https://github.com/alpenlabs/bitcoin-bosd/pull/31/commits/036ce0e369e4c3ed03578b1f770bff709468d490)